addon/components/sl-split-grid.js

```diff
@@ -0,0 +1,564 @@
+     *
+     * Value is either "ascending" or "descending".
+     *
+     * @property {string} sortedDirection
+     * @default  null
+     */
+    sortedDirection: null,
+
+    // -------------------------------------------------------------------------
+    // Observers
+
+    /**
+     * Does cleanup for internal state when content length has changed
+     *
+     * @function handleNewContent
+     * @observes content.length
+     * @returns  {void}
+     */
+    handleNewContent: function() {
+        this.set( 'pendingData', false );
+
+        if ( !this.get( 'hasMorePages' ) ) {
+            this.disableContinuousPaging();
+        }
+    }.observes( 'content.length' ),
+
+    /**
+     * Resize the split-grid's detail pane content to the set height value
+     *
+     * @function resizeDetailContent
+     * @observes autoHeight, detailContentHeight, "didInsertElement" event
+     * @returns  {void}
+     */
+    resizeDetailContent: function() {
+        this.$( '.detail-pane .content' ).height( this.get( 'detailContentHeight' ) );
+    }.observes( 'detailContentHeight' ).on( 'didInsertElement' ),
+
+    /**
+     * Resize the split-grid's list pane content to the set height value
+     *
+     * @function resizeListContent
+     * @observes autoHeight, listContentHeight, "didInsertElement" event
+     * @returns  {void}
+     */
+    resizeListContent: function() {
+        this.$( '.list-pane .content' ).height( this.get( 'listContentHeight' ) );
+    }.observes( 'listContentHeight' ).on( 'didInsertElement' ),
+
+    /**
+     * Setup the "continuous paging" functionality, if the data set is not complete
+     *
+     * @function setupContinuousPaging
+     * @observes "didInsertElement" event
+     * @returns  {void}
+     */
+    setupContinuousPaging: function() {
+        if ( this.get( 'hasMorePages' ) ) {
+            this.enableContinuousPaging();
+        }
+    }.on( 'didInsertElement' ),
+
+    /**
+     * Setup the auto resize of content height, or set the hard-coded height in pixels
+     *
+     * @function setupSizes
+     * @observes "didInsertElement" event
+     * @returns  {void}
+     */
+    setupSizes: function() {
+        if ( this.get( 'autoHeight' ) === true ) {
+            Ember.$( window ).on( 'resize', this.updateContentHeight.bind( this ) );
+            this.updateContentHeight();
+        } else {
+            this.resizeDetailContent();
+            this.resizeListContent();
+        }
+    }.on( 'didInsertElement' ),
+
+    // -------------------------------------------------------------------------
+    // Methods
+
+    /**
+     * The text for the actions column's `style` attribute
+     *
+     * @function actionsColumnStyle
+     * @observes actionsColumnWidth
+     * @returns  {string} - The `style` attribute for the column
+     */
+    actionsColumnStyle: function() {
+        return 'width: ' + this.get( 'actionsColumnWidth' ) + 'px;';
+    }.property( 'actionsColumnWidth' ),
```

Comment:
Is it anticipated or designed that the "actionsColumnWidth" property will be assigned its value via configuration or changed during the running of the application?  I do not see it listed as a property at #/demos/sl-split-grid.  If this is not the intention I'd prefer to see this defined as a LESS variable and baked into the CSS so that a developer can override if necessary and we can realize some performance gain.